src/components/card/ProfileCard.jsx

```
@@ -0,0 +1,36 @@
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)